### PR TITLE
let and id instead of var and class for WebAssembly example

### PR DESCRIPTION
**Pull request by christopher-besch**
Using a cleaner and more modern example for [Compiling a New C/C++ Module to WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly/C_to_wasm#calling_a_custom_function_defined_in_c).

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/webassembly/c_to_wasm/index.md [hunk 1]
```diff
@@ -144,22 +144,21 @@ If you have a function defined in your C code that you want to call as needed fr
...
 
     ```html
-    <button class="mybutton">Run myFunction</button>
+    <button id="mybutton">Run myFunction</button>
     ```
 
 7. Now add the following code at the end of the first {{HTMLElement("script")}} element:
 
     ```js
-    document.querySelector('.mybutton')
```
Comment (Josh-Cena) [inline files/en-us/webassembly/c_to_wasm/index.md:153]: AFAICT—`querySelector` _is_ the more modern alternative to `getElementById`.

Comment (teoli2003) [reply files/en-us/webassembly/c_to_wasm/index.md:153]: It makes more sense to have an id here than a class.

Using `querySelector("#mybutton")` or `queryElementById("myButton")` is not important.

File: files/en-us/webassembly/c_to_wasm/index.md [hunk 2]
```diff
@@ -144,22 +144,21 @@ If you have a function defined in your C code that you want to call as needed fr
...
-            alert('check console');
-            var result = Module.ccall(
-                'myFunction',  // name of C function
-                null,  // return type
-                null,  // argument types
-                null  // arguments
-            );
-        });
+    document.getElementById("mybutton").onclick = () => {
+        alert('check console');
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/webassembly/c_to_wasm/index.md:154]: Let's keep it for the moment, then. (Note that `alert()` in MDN live samples are just ignored, …)

File: files/en-us/webassembly/c_to_wasm/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/webassembly/c_to_wasm/index.md:153]: Also I think we would prefer `addEventListener` to `onclick`?

Comment (teoli2003) [reply files/en-us/webassembly/c_to_wasm/index.md:153]: I have no strong feelings here, there are so many `onXYZ`…

For the moment, I accept both (even if `addEventListener` has some advantages, it is sometimes more difficult to read)

**Review by teoli2003 — APPROVED** [review]
👍 (I leave you to merge it, once you think it is ready)

File: files/en-us/webassembly/c_to_wasm/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/webassembly/c_to_wasm/index.md:153]: On https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener we seem to give our own preference towards `addEventListener`. And given it's already used here before, I would suggest keeping it as-is.

**Review by Josh-Cena — APPROVED** [review]
Thank you!

**Pull request merged**